connect-examples/oauth/node/index.js

```diff
@@ -0,0 +1,80 @@
+/* This sample demonstrates a bare-bones implementation of the Square Connect OAuth flow in Node.js:
```

Comment:
Add a copyright header to this file:

/*
Copyright 2020 Square Inc.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/